src/parsers/pm2ps_parser.jl

```diff
@@ -248,23 +254,7 @@ function make_renewable_fix(gen_name, d, bus)
 end
 
 function make_generic_battery(gen_name, d, bus)
-
-    # TODO: placeholder
-    #battery=GenericBattery(;
-    #    name=gen_name,
-    #    available=Bool(d["gen_status"]),
-    #    bus=bus,
-    #    energy=,
-    #    capacity=,
-    #    rating=,
-    #    activepower=,
-    #    inputactivepowerlimits=,
-    #    outputactivepowerlimits=,
-    #    efficiency=,
-    #    reactivepower=,
-    #    reactivepowerlimits=,
-    #)
-    #return battery
+    error("Not implemented yet.")
```

Comment:
Someone is going to have to re-type most of the code that you removed.  Might as well leave it.

Reply:
> Don't use block comments for Julia code. Prefer using the # prefix. If you are commenting code, consider deleting it instead.

We had something like this in our style guide. This isn't a significant amount of code, we can rewrite it without mistakes and without much effort if needed in the future.